go.mod

```diff
@@ -35,3 +35,5 @@ require (
 )
 
 replace github.com/gorilla/rpc v1.2.0+incompatible => github.com/gorilla/rpc v1.2.0
+
+replace github.com/minio/minio-go/v6 => ../minio-go
```

Comment:
Intended?

Reply:
Yes, until `minio-go` with the new listing API is released. This is to test the PR